acf-woocommerce-product-type-locations.php

```diff
@@ -18,13 +18,11 @@
 
 // Location Rules
 add_filter('acf/location/rule_types', 'wc_product_acf_location_rule_types', 50, 1); 
-add_filter('acf/location/rule_values/woocommerce_product_type', 'wc_product_acf_location_rule_types_woocommerce_product_type', 50, 1);
```

Comment:
We don't rule for Woocoomerce Product Type Eg Simple, Grouped, Etc. has Post Taxonomy Tag(Product_Type) option is in ACF.